src/main/java/core/basesyntax/model/machine/Car.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax.model.machine;
 
+import javax.persistence.Entity;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+
+@Entity
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
```

Comment:
we don't need this annotation here, only in Machine class

Reply:
thanks. fixed.